packages/agents/sdk/src/lib/helpers/shared.ts

```diff
@@ -55,3 +55,59 @@ export const parseConnextLog = (log: providers.Log): any => {
     throw new ParseConnextLogFailed({ log });
   }
 };
+
+export type Transfer = {
+  transfer_id: string;
+  nonce: number;
+  to: string;
+  call_data: string;
+  origin_domain: string;
+  destination_domain: string;
+  receive_local: boolean;
+  origin_chain: string;
+  origin_transacting_asset: string;
+  origin_transacting_amount: string;
+  origin_bridged_asset: string;
+  origin_bridged_amount: string;
+  xcall_caller: string;
+  xcall_transaction_hash: string;
+  xcall_timestamp: string;
+  xcall_gas_price: string;
+  xcall_gas_limit: string;
+  xcall_block_number: string;
+  destination_chain: string;
+  status: string;
+  routers: string[];
+  destination_transacting_asset: string;
+  destination_transacting_amount: string;
+  destination_local_asset: string;
+  destination_local_amount: string;
+  execute_caller: string;
+  execute_transaction_hash: string;
+  execute_timestamp: number;
+  execute_gas_price: string;
+  execute_gas_limit: string;
+  execute_block_number: number;
+  execute_origin_sender: string;
+  reconcile_caller: string;
+  reconcile_transaction_hash: string;
+  reconcile_timestamp: string;
+  reconcile_gas_price: string;
+  reconcile_gas_limit: string;
+  reconcile_block_number: number;
+  update_time: string;
+  delegate: string;
+  message_hash: string;
+  canonical_domain: string;
+  slippage: number;
+  origin_sender: string;
+  bridged_amt: string;
+  normalized_in: string;
+  canonical_id: string;
+  router_fee?: string;
+  xcall_tx_origin: string;
+  execute_tx_origin: string;
+  reconcile_tx_origin: string;
+  relayer_fee: string;
+  error_status?: string;
```

Comment:
execute_simulation_input?: string,
    execute_simulation_from?: string,
    execute_simulation_to?: string,
    execute_simulation_network?: string,
    error_message?: string,
    message_status?: string,
    relayer_fees?: string[]

Reply:
added the ones i think was missing, @coinhippo review is required here as well.

Reply:
@nrsirapop @pjenviri Can we check if all the types are present which we are using or need.

Reply:
for list ref: https://github.com/connext/monorepo/blob/0ebcfe1fdf6a2687eecf7de304c46df7ea4b0b89/packages/adapters/database/db/schema.sql

Reply:
Added the missing ones.